cmd/backfill-redis/main.go

```diff
@@ -210,6 +208,30 @@ func main() {
 	}
 }
 
+func redisClient() *redis.Client {
+
+	// #nosec G402
+	tlsConfig := &tls.Config{
+		InsecureSkipVerify: *insecureSkipVerify,
+	}
+
+	if *enableTLS {
+		return redis.NewClient(&redis.Options{
+			Addr:      fmt.Sprintf("%s:%s", *redisHostname, *redisPort),
+			Password:  *redisPassword,
+			Network:   "tcp",
+			TLSConfig: tlsConfig,
+			DB:        0, // default DB
+		})
+	}
+	return redis.NewClient(&redis.Options{
+		Addr:     fmt.Sprintf("%s:%s", *redisHostname, *redisPort),
+		Password: *redisPassword,
+		Network:  "tcp",
+		DB:       0, // default DB
+	})
+}
```

Comment:
minor nit here and in api.go as well: could you reduce the amount of duplicated code here?

for example:
```
opts := &redis.Options{
	Addr:      fmt.Sprintf("%s:%s", *redisHostname, *redisPort),
	Password:  *redisPassword,
	Network:   "tcp",
	DB:        0, // default DB
}

if *enableTLS {
	opts.TLSConfig = &tls.Config{
		InsecureSkipVerify: *insecureSkipVerify,
	}
}

return redis.NewClient(opts)
```

Reply:
Pushed a commit ( https://github.com/sigstore/rekor/pull/1998/commits/191f24a961b8a2516286bca0241ea78df3df8fcf ) to reduce duplicate code.